stix_shifter_modules/crowdstrike_logscale/configuration/config.json

```diff
@@ -0,0 +1,27 @@
+{
+    "connection": {
```

Comment:
Can you add a page_size option to the connection?

Reply:
This connector does not need page size parameter, sine the around parameter of the api call can fetch as many records ,whatever the length may be

Reply:
I disagree. Paging is a standard function when dealing with API's that can return large datasets. Not implementing paging means you are risking memory and performance issues. The current implementation does not address the original concern. The purpose behind paging (and the around parameter) is to break a large request (10,000 results as an example) into smaller, more manageable request. My understanding is that the current implementation would request 200 results, then follow up with a request for 9,800 results. Which would defeat the purpose of implementing the around function in the first place.

Reply:
The  code has been updated with Pagination as suggested.